cpu/stm32f1/include/stm32f103xe.h

```diff
@@ -1767,6 +1767,10 @@ typedef struct
 
 #define AFIO_MAPR_TIM4_REMAP                 ((uint32_t)0x00001000)        /*!< TIM4_REMAP bit (TIM4 remapping) */
 
+/*!< TIM4_REMAP configuration */
```

Comment:
These are the only two missing... I guess I could place them somewhere else though